resources/views/instrument/show.blade.php

```diff
@@ -59,13 +59,44 @@
 
             <!-- Card Footer with Delete Button -->
             <div class="card-footer text-muted text-center">
+                <!-- Delete Form -->
                 <form action="{{ route('instrument.delete', $viewData['instrument']->getId()) }}" method="POST">
                     @csrf
                     @method('DELETE')
                     <button type="submit" class="btn btn-danger">{{ __('messages.delete_instrument') }}</button>
                 </form>
+
+                <!-- Add to Cart Form -->
+                <!-- Add to Cart Form -->
+            <form id="addToCartForm" action="{{ route('instrument.addToCart', ['id' => $viewData['instrument']->id]) }}" method="POST">
+                @csrf
+                <div class="mb-3">
+                    <label for="quantity" class="form-label">{{ __('attributes.quantity') }}</label>
+                    <input type="number" id="quantity" name="quantity" class="form-control" min="1" max="{{ $viewData['instrument']->getQuantity() }}" value="1" required data-max-quantity="{{ $viewData['instrument']->getQuantity() }}">
+                </div>
+                <button type="submit" class="btn btn-primary">{{ __('messages.add_to_cart') }}</button>
+            </form>
```

Comment:
I suggest creating a template for this inside cart, and then importing. Template could be -> views/cart/"add.blade.php"